minecraft@.service

```
@@ -48,10 +48,25 @@ ExecStart=/bin/sh -c \
           -Xms${MCMINMEM} \
           -Xmx${MCMAXMEM} \
           -XX:+UseG1GC \
-          -XX:+CMSClassUnloadingEnabled \
-          -XX:ParallelGCThreads=2 \
-          -XX:MinHeapFreeRatio=5 \
-          -XX:MaxHeapFreeRatio=10 \
+          -XX:+ParallelRefProcEnabled \
+          -XX:MaxGCPauseMillis=200 \
+          -XX:+UnlockExperimentalVMOptions \
+          -XX:+DisableExplicitGC \
```

Disabling explicit gc entirely could lead to some already badly behaving code to bail out entirely. It also destroys reasonable usages of explicit gc invocations, how about this (it still prevents stop the world issues, but with less risks):
```suggestion
          -XX:+ExplicitGCInvokesConcurrent \
          -XX:+CMSClassUnloadingEnabled \
```